packages/solid-repl/src/components/repl.tsx

```diff
@@ -123,7 +123,7 @@ export const Repl: ReplProps = (props) => {
     if (event === 'ROLLUP') {
       const currentMap = { ...importMap() };
       for (const file in currentMap) {
-        if (!(file in compiled) && currentMap[file] === `https://jspm.dev/${file}`) {
+        if (!(file in compiled) && currentMap[file] === `https://esm.sh/${file}`) {
```

Comment:
Could also be worth removing items from the import map if the URL matches the `https://jspm.dev/${file}`, so that current users are also updated to esm.sh.

This comes with the risk of updating users whose projects depend on the jspm URL, but this seems unlikely.
Happy to change if needed :)

Reply:
If we store the import map in saved repls then we should upgrade everything to esm.sh